spec.html

```diff
@@ -4385,7 +4381,28 @@ <h1>Runtime Semantics: MV</h1>
               The MV of <emu-grammar>StrUnsignedDecimalLiteral ::: DecimalDigits ExponentPart</emu-grammar> is the MV of |DecimalDigits| times 10<sup>_e_</sup>, where _e_ is the MV of |ExponentPart|.
             </li>
           </ul>
-          <p>Once the exact MV for a String numeric literal has been determined, it is then rounded to a value of the Number type. If the MV is 0, then the rounded value is *+0*<sub>𝔽</sub> unless the first non white space code point in the String numeric literal is `-`, in which case the rounded value is *-0*<sub>𝔽</sub>. Otherwise, the rounded value must be the Number value for the MV (in the sense defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>), unless the literal includes a |StrUnsignedDecimalLiteral| and the literal has more than 20 significant digits, in which case the Number value may be either the Number value for the MV of a literal produced by replacing each significant digit after the 20th with a 0 digit or the Number value for the MV of a literal produced by replacing each significant digit after the 20th with a 0 digit and then incrementing the literal at the 20th digit position. A digit is significant if it is not part of an |ExponentPart| and</p>
+          <p>For the MV of any production not shown above, see <emu-xref href="#sec-static-semantics-mv"></emu-xref>.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-stringtonumber" aoid="StringToNumber">
+          <h1>StringToNumber ( _str_ )</h1>
+          <p>The abstract operation StringToNumber takes argument _str_ (a String) and returns a Number. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Let _text_ be ! StringToCodePoints(_str_).
+            1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
+            1. If _literal_ is a List of errors, return *NaN*.
+            1. If _literal_ contains a |StrUnsignedDecimalLiteral| and _literal_ has more than 20 significant digits, then
+              1. Let _lit_ be an implementation-defined choice of:
+                * a literal produced from _literal_ by replacing each significant digit after the 20th with a `0` digit
+                * a literal produced from _literal_ by replacing each significant digit after the 20th with a `0` digit and then incrementing the literal at the 20th significant digit position
```

Comment:
It's probably worth having the NOTE below be reproduced here as well.

Reply:
Done.